source/common/upstream/load_balancer_impl.h

```diff
@@ -806,17 +806,17 @@ class LoadBalancerSubsetInfoImpl : public LoadBalancerSubsetInfo {
   bool listAsAny() const override { return list_as_any_; }
 
 private:
-  const bool enabled_;
   const envoy::config::cluster::v3::Cluster::LbSubsetConfig::LbSubsetFallbackPolicy
       fallback_policy_;
-  const envoy::config::cluster::v3::Cluster::LbSubsetConfig::LbSubsetMetadataFallbackPolicy
-      metadata_fallback_policy_;
   const ProtobufWkt::Struct default_subset_;
   std::vector<SubsetSelectorPtr> subset_selectors_;
-  const bool locality_weight_aware_;
-  const bool scale_locality_weight_;
-  const bool panic_mode_any_;
-  const bool list_as_any_;
+  const envoy::config::cluster::v3::Cluster::LbSubsetConfig::LbSubsetMetadataFallbackPolicy
```

Comment:
I see this field moved but not `fallback_policy_`, why?
Is there added benefit by moving these enum types?

Reply:
Good catch, I just forgot to pull the other enum down. Technically, moving it does doesn't save memory but I feel it'll be better for consistency to move them down anyway.